src/main/resources/pom-servers.xml

```diff
@@ -132,44 +132,32 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>net.wasdev.wlp.maven.plugins</groupId>
+                        <groupId>io.openliberty.tools</groupId>
                         <artifactId>liberty-maven-plugin</artifactId>
                         <version>${openliberty.maven.version}</version>
-
-                        <configuration>
-                            <assemblyArtifact>
-                                <groupId>io.openliberty</groupId>
-                                <artifactId>openliberty-runtime</artifactId>
-                                <version>${openliberty.version}</version>
-                                <type>zip</type>
-                            </assemblyArtifact>
-
-                            <configFile>src/main/liberty/config/server.xml</configFile>
+                        <executions>
+                            <execution>
+                            <id>package-server</id>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>create</goal>
+                                <goal>deploy</goal>
+                                <goal>package</goal>
+                            </goals>
+                            <configuration>
+                                <outputDirectory>target/wlp-package</outputDirectory>
+                            </configuration>
+                            </execution>
+                        </executions>
+                        <configuration>   
                             <appArchive>${project.build.directory}/${final.name}.war</appArchive>
-                            <packageFile>${project.build.directory}/${final.name}.jar</packageFile>
-                            <include>runnable</include>
+                            <packageType>runnable</packagType>
```

Comment:
Typo in end tag

Reply:
it should be <packageType>jar</packageType>